spec/new-productions.htm

```diff
@@ -9,6 +9,18 @@
   </emu-grammar>
 </emu-clause>
 
+<emu-clause id="public-field-definition-early-errors">
+  <h1>Static Semantics: Early Errors</h1>
+    PublicFieldDefinition :
+      PropertyName[?Yield] Initializer?
+  <emu-grammar>
+  </emu-grammar>
+  <emu-alg>
+    1. It is a SyntaxError if |Initializer| contains |NewTarget|.
```

Comment:
See the definition (and usage) of `Contains` in this section: https://tc39.github.io/ecma262/#sec-static-semantic-rules